README.md

```diff
@@ -90,6 +90,13 @@ rke2_kubevip_ipvs_lb_enable: false
 # Enable layer 4 load balancing for control plane using IPVS kernel module
 # Must use kube-vip version 0.4.0 or later
 
+rke2_kubevip_service_election_enable: false
+# By default ARP mode provides a HA implementation of a VIP (your service IP address) which will receive traffic on the kube-vip leader.
+# To circumvent this kube-vip has implemented a new function which is "leader election per service", 
```

Comment:
Please remove the trailing white space from this line. Thanks
https://github.com/lablabs/ansible-role-rke2/actions/runs/7888640859/job/21870718938?pr=192